### DISCUSSION TITLE
[Auth0 client] Tokens

### DISCUSSION THREAD

**Discussion by georgev-97** [mia-platform/community#367] [Q&A] [answered]
### Which section?

Console

### Which is the version?

12

### Description

Good morning,
I am currently using the version 3.3.2 of the Auth0 client, I followed the documentation in its integrity to configure both the client and the auth0 tenant.
The `GET-POST /users` works fine while the `POST /oauth/token` works only partially, I can confirm that the request is being processed correctly by the tenant as I checked the logs on the auth0 dashboard, but I always get an empty response 200 instead of the tokens.
What could be the reason?

**Comment by georgev-97** [comment]
In addition, if I make the same request directly to auth0 I correctly receive both the `access_token` and the `id_token`

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @georgev-97, does the client in your configuration use the `website` scope? If so tokens are not sent through the body but converted in a session that you can find in the `Set-Cookie` response header

**Reply by georgev-97** [reply]
Yes, this was definitely the problem, thank you!

**Reply by fredmaggiowski** [reply]
you are welcome, I think that the doc is wrong on some parts. @malta895 let's take a look and improve it, please

**Reply by georgev-97** [reply]
If I configure multiple clients, how do I make requests to the one I want instead of the default? What is the field in the request that specifies the client?

**Reply by malta895** [reply]
Hi @georgev-97,
for that you can use the `connection` field. It is explained in details how to do so [in this doc section](https://docs.mia-platform.eu/docs/runtime_suite/auth0-client/overview_and_usage#how-to-configure-multiple-environment)
